bridge/xmpp/xmpp.go

```diff
@@ -331,7 +331,7 @@ func (b *Bxmpp) skipMessage(message xmpp.Chat) bool {
 	}
 
 	// skip empty messages
-	if message.Text == "" {
+	if message.Text == "" &&  message.Chatstate == "" {
```

Comment:
If you want to change this, please open a new PR for this change (it's not related to the QuoteLengthLimit feature)